include/spirv-tools/optimizer.hpp

```diff
@@ -838,6 +838,14 @@ Optimizer::PassToken CreateWrapOpKillPass();
 // capabilities.
 Optimizer::PassToken CreateAmdExtToKhrPass();
 
+// Replaces the internal version of GLSLstd450 OpInterpolateAt* instructions
```

Comment:
Calling the instructions `OpInterpolateAt*` is confusing because they are not instruction opcodes.  When I first saw this, I searched for `OpInterpolateAt` in the specs to try to find them, but I could not find it.  We should say the "GLSLstd450 `InterpolateAt*` extended instructions" or something similar to that.

Reply:
OK.